docs/extensibility/display-widgets.md

```diff
@@ -19,6 +19,10 @@ Text widgets render values as a simple text. This is the default behavior for al
 }
 ```
 
+<p>
+<img src="./assets/display-widgets/Text.png" alt="Example of a text widget" style="max-width:20%;">
+</p>
```

Comment:
is adding images as html really that necessary? Wouldn't it be better to just use markdown syntax?
```suggestion
![Example of a text widget](./assets/display-widgets/Text.png)
```
sure, we lose the `style=max-width`, but do we really need it that much to mess up the .md code?

Reply:
this comment of course applies to ALL the images, not only this one

Reply:
Also, did you check if git supports extended syntax?
maybe this will work, or github has a similar equivalent?
```
![Example of a text widget](./assets/display-widgets/Text.png){max-width:20%}
```

Reply:
The <img> tags are used all over the md files. They might not be the default ones, but they are helpful in this situation 
1. we can set the width to 20 (what is impossible with just md, and looks ugly without it) 
2. I've also added borders because it turned out to be the requirement.